src/foam/u2/stack/DesktopStackView.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.stack',
+  name: 'DesktopStackView',
+  extends: 'foam.u2.stack.StackView',
+
+  requires: [
+    'foam.nanos.controller.Memento',
+    'foam.u2.dialog.Popup',
+    'foam.u2.stack.Stack'
+  ],
+
+  properties: [
+    {
+      name: 'popupsOpened',
+      documentation: 'Determines of a popup is open at a specific index',
+      factory: () => ({})
+    }
+  ],
+
+  css: '%CUSTOMCSS%',
+
+  methods: [
+    // TODO: Why is this init() instead of initE()? Investigate and maybe fix.
```

Comment:
What is this comment for?

Reply:
@kgrgreer fixed - this was a clipboard artifact